paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -162,19 +190,11 @@ def spotfleet_metrics_provider(spotfleet_request_id, resource, pool_settings):
         for pair in zip(free_pool_resources, total_pool_resources)
     ])
     target_utilization = pool_settings.get('target_utilization', DEFAULT_TARGET_UTILIZATION)
-    error = utilization - target_utilization
-    current, target = get_spot_fleet_delta(resource, error)
-    return current, target
+    return utilization - target_utilization
 
 
 def get_spot_fleet_delta(resource, error):
-    ec2_client = boto3.client('ec2', region_name=resource['region'])
-    spot_fleet_request = ec2_client.describe_spot_fleet_requests(
-        SpotFleetRequestIds=[resource['id']])['SpotFleetRequestConfigs'][0]
-    if spot_fleet_request['SpotFleetRequestState'] != 'active':
-        raise ClusterAutoscalingError('Can not scale non-active spot fleet requests. This one is "%s"' %
-                                      spot_fleet_request['SpotFleetRequestState'])
-    current_capacity = int(spot_fleet_request['SpotFleetRequestConfig']['TargetCapacity'])
+    current_capacity = int(resource['sfr']['SpotFleetRequestConfig']['FulfilledCapacity'])
```

Comment:
This does sound correct, except when a SFR is just getting started, or during times of low EC2 capacity, the autoscaler is going to "overshoot" right? Because on scaleup it will only look at what is fulfilled *so-far*, and then scale up even more?